pandora-client-web/src/components/Eula/privacyPolicy.tsx

```diff
@@ -75,8 +75,10 @@ export function PrivacyPolicyContent(): ReactElement {
 				</li>
 			</ul>
 			<p>We do not share Your personal information with third parties.</p>
-			<h3>How do we store the data?</h3>
-			<p>We will retain the aforementioned data potentially indefinitely so we can fulfill the purposes described in this Privacy Policy; typically for the lifetime of Your Pandora account.</p>
+			<h3>How do We store the data?</h3>
+			<p>We will retain the aforementioned data potentially indefinitely so We can fulfill the purposes described in this Privacy Policy; typically for the lifetime of Your Pandora account.</p>
+			<p>If You do not log into Your Pandora account during a full calendar year, We will delete Your Pandora account and all the data stored in it during the following year.</p>
```

Comment:
Should it be only a single year or should we allow more?

Reply:
We can allow more, if that is fine with us and the GDPR.

Reply:
actually, I think GDPR does not require us to delete user accounts after some inactivity, so it is more like for us to free up space in database

Reply:
Reverted the new text and opened issue:
https://github.com/Project-Pandora-Game/pandora/issues/610